apps/scratch/src/pages/tests/Edit.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import { readFileSync } from "fs";
+import { resolve } from "path";
+import { test, expect } from "playwright-test-coverage";
+
+// eslint-disable-next-line no-undef
+const testTask = readFileSync(resolve(__dirname, "test-task.zip"));
+
+declare global {
+  interface Window {
+    postedMessages: { message: unknown; options: unknown }[];
+  }
+}
+
+test.describe("/edit/taskId", () => {
+  test.beforeEach(async ({ page, baseURL }) => {
+    page.on("framenavigated", async () =>
+      page.evaluate(() => {
+        window.postedMessages = [];
+
+        // @ts-expect-error - we mock the parent window
+        window.parent = {
+          postMessage: (message, options) => {
+            window.postedMessages.push({ message, options });
+          },
+        };
+      }),
+    );
+
+    await page.goto(`${baseURL!}/edit/some-task-id`);
+
+    await page.waitForSelector("#root");
+  });
+
+  test("can get height via window.postMessage", async ({ page }) => {
+    await page.evaluate(() => {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const event = new Event("message") as any;
```

Comment:
Is there no proper typing ? 
Should we get inspired by https://github.com/DerZade/typescript-event-target ? It seems like it might be relevant to have properly typed events :-)